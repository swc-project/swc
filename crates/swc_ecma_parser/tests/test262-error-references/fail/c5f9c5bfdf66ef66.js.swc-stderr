  x Expected ';', '}' or <eof>
   ,-[$DIR/tests/test262-parser/fail/c5f9c5bfdf66ef66.js:1:1]
 1 | a if;
   : | ^^
   : `-- This is the expression part of an expression statement
   `----
