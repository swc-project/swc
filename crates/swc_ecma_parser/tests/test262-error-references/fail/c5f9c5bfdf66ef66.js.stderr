
  x Expected ';', '}' or <eof>
   ,-[$DIR/tests/test262-parser/fail/c5f9c5bfdf66ef66.js:1:1]
 1 | a if;
   :   ^^
   `----

Error: 
  > This is the expression part of an expression statement
   ,-[$DIR/tests/test262-parser/fail/c5f9c5bfdf66ef66.js:1:1]
 1 | a if;
   : ^
   `----
